src/main/java/com/blackduck/integration/detect/lifecycle/boot/DetectBoot.java

```diff
@@ -182,24 +182,8 @@ public Optional<DetectBootResult> boot(String detectVersion, String detectBuildD
         }
         logger.debug("Main boot completed. Deciding what Detect should do.");
 
-        if (detectArgumentState.isGenerateAirGapZip()) {
-            try {
-                AirGapType airGapType = new AirGapTypeDecider().decide(detectArgumentState);
-                AirGapCreator airGapCreator = detectBootFactory
-                    .createAirGapCreator(
-                        detectConfigurationFactory.createConnectionDetails(),
-                        detectConfigurationFactory.createDetectExecutableOptions(),
-                        freemarkerConfiguration,
-                        installedToolManager,
-                        installedToolLocator
-                    );
-
-                File airGapZip = airGapCreator.createAirGapZip(airGapType, directoryManager.getRunHomeDirectory());
-
-                return Optional.of(DetectBootResult.exit(propertyConfiguration, airGapZip, directoryManager, diagnosticSystem));
-            } catch (DetectUserFriendlyException e) {
-                return Optional.of(DetectBootResult.exception(e, propertyConfiguration, directoryManager, diagnosticSystem));
-            }
+        if (detectArgumentState.isGenerateAirGapZip()) { // option 1
```

Comment:
this comment confuses me, is it adding value?

Reply:
Committed this comment as mistake (was just leaving myself some notes as I was reading through the code in preparation to make sonar related changes) 

Removed!